sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -112,7 +112,13 @@ def __init__(self, name, init_config, instances):
         self.ad_last_check = 0
 
         self.proc = self.instance.get('stored_procedure')
-        self.proc_type_mapping = {'gauge': self.gauge, 'rate': self.rate, 'histogram': self.histogram}
+        self.proc_type_mapping = {
+            "gauge": self.gauge,
+            "rate": self.rate,
+            "histogram": self.histogram,
+            "count": self.count,
+            "monotonic_count": self.monotonic_count,
```

Comment:
do we have any test coverage for this at all? It might be worth adding some tests if not?